integration/walletmock/wallet.go

```diff
@@ -1,93 +0,0 @@
-package walletmock
```

Comment:
qq, I can't figure out where the logic that was here went

Reply:
I was pretty much l2 tx creation methods, that were moved to the tx injector.